### PR TITLE
Dev 317 fix new user check

### PR DESCRIPTION
**Pull request by mejiasd3v**

### CODE REVIEW

**Review by Thehitesh172 — APPROVED** [review]

**Review by parth-15 — APPROVED** [review]

**Review by royvardhan — APPROVED** [review]

File: test/Facets/FeeCollectorFacet.t.sol [hunk 1]
```diff
@@ -78,13 +78,12 @@ contract FeeCollectorFacetTest is DiamondTest {
...
         _mintAndDeposit(stratosphereMemberBasic, 1000);
         _fundUserWithfeeToken(stratosphereMemberBasic, boostFeeLvl1);
+        vm.expectRevert(BoostFacet__BoostAlreadyClaimed.selector);
         boostFacet.claimBoost(1);
         vm.stopPrank();
 
         vm.startPrank(makeAddr("diamondOwner"));
         feeCollectorFacet.collectBoostFees();
-
-        assertEq(feeToken.balanceOf(depositFeeReceiver1) + feeToken.balanceOf(depositFeeReceiver2), boostFeeLvl1);
```
Comment (Thehitesh172) [inline test/Facets/FeeCollectorFacet.t.sol:87]: ummm.. that expectRevert is stopping us from testing fee collection

File: test/Facets/FeeCollectorFacet.t.sol [hunk 2]
```diff
@@ -78,13 +78,12 @@ contract FeeCollectorFacetTest is DiamondTest {
         vm.startPrank(stratosphereMemberBasic);
         _mintAndDeposit(stratosphereMemberBasic, 1000);
         _fundUserWithfeeToken(stratosphereMemberBasic, boostFeeLvl1);
+        vm.expectRevert(BoostFacet__BoostAlreadyClaimed.selector);
```
Comment (Thehitesh172) [inline test/Facets/FeeCollectorFacet.t.sol:81]:
```suggestion
        vm.warp(block.timestamp + 2 days);
```

File: test/Facets/BoostFacet.t.sol [hunk 3]
```diff
@@ -77,7 +77,7 @@ contract BoostFacetTest is DiamondTest {
     function test_RevertIf_NotStratosphereMember() public {
         vm.startPrank(user);
         _mintAndDeposit(user, testDepositAmount);
-        vm.expectRevert(BoostFacet__InvalidBoostLevel.selector);
+        vm.expectRevert(BoostFacet__BoostAlreadyClaimed.selector);
```
Comment (Thehitesh172) [inline test/Facets/BoostFacet.t.sol:80]:
```suggestion
        vm.warp(block.timestamp + 2 days);
        vm.expectRevert(BoostFacet__InvalidBoostLevel.selector);
```

**Comment by Thehitesh172** [comment]
@Parth https://github.com/VaporFi/liquid-mining/blob/b85ea7c4b3e02d202426aa9f0e113f1527ef92e5/src/facets/DepositFacet.sol#L67 and https://github.com/VaporFi/liquid-mining/blob/b85ea7c4b3e02d202426aa9f0e113f1527ef92e5/src/facets/BoostFacet.sol#L41

**Pull request merged**